src/test/suite/fixtures/recorded/actions/clearSource2.yml

```diff
@@ -0,0 +1,29 @@
+languageId: plaintext
```

Comment:
Not sure how this test differs from the other "source" test?

Reply:
One test clone up the other one tests clone down. The implementation of source is different

Reply:
Again, I'd argue that the "source" output of the test itself captures that. I'd argue we just need one or two tests to check that it is possible to use "source" as a mark